plugins/modules/net_tools/nmcli.py

```diff
@@ -295,7 +295,7 @@
             - This is only used with 'bridge-slave' - 'hairpin mode' for the slave, which allows frames to be sent back out through the slave the
               frame was received on.
         type: bool
-        default: yes
+        default: no
```

Comment:
This is a breaking change and thus cannot be merged.

You could adjust the documentation to point out this difference to the CLI version, but changing this will break existing playbooks and roles that potentially rely on our default value.

Reply:
The nmcli module has tons of options, so I think even if its added to the documentation it will not get read.

In fact you will only need hairpin mode in some very specific cases, so its logic its defaulted to false.
So I really think we should bite the bitter pill and change this to default to no.

Also having hairpin enabled is not something that just gives you additional features or so, but it can seriously break things on the network (like create some loops etc).

So yea its a breaking change (adjusted the changelog for this), but I think we should just do it :)

Reply:
Well, we still do not accept breaking changes (except for a few very important reasons, and this is not one of them). See https://github.com/ansible-collections/community.general/issues/582 for our versioning and deprecation policy.

To change this, the current default needs to be deprecated (with a deprecation warning printed) for a certain amount of time until it can change.